src/Makefile.am

```diff
@@ -512,7 +511,6 @@ libqueso_include_HEADERS += contrib/getpot/getpot.h
 
 nodist_libqueso_include_HEADERS =
 nodist_libqueso_include_HEADERS += $(top_builddir)/config_queso.h
-nodist_libqueso_include_HEADERS += $(top_builddir)/src/core/inc/queso.h
```

Comment:
I'm a little confused.  Not by what you did, but by what we were doing.   It seems that before this change, neither `queso.h` nor `config_queso.h` were shipped in our tarballs.  If that's the case (and I will check that), then it means users could never `#include` these files anyway.